### ISSUE TITLE
Error: sh: wp: command not found

### ISSUE THREAD

**Issue by LorincJuraj** [versionpress/support#81]
HI.

I have installed VP on my shared cPanel hosting. It was a pain to make git and the WP-CLI work, but it finally works now. I had a problem that when I cloned my repo (from Bitbucket) and try to do  `wp vp restore-site` I was getting error `Error: sh: wp: command not found`. I was able to solve it by changing following line in `VPCommandUtils.php`

```
$cliCommand = "wp $command";
```
and replacing it with 
```
$cliCommand = "/home/myusername/wp-cli.phar $command";
```
Strange thing is that I am able to use wp-cli ajust by typing `wp` into terminal (ssh) but VP is not able to access it. Do you have any idea why?

I know this might not be exactly VP issue but is very related to it and others can come across it too. If you thing it doesn't belong here feel free to delete it.

Here are my system details:

```
Quick summary
array (
  'wordpress-version' => '4.7.1',
  'versionpress-version' => '3.0.2',
  'operating-system' => 'Linux lhcp1050.webapps.net 2.6.32-673.26.1.lve1.4.20.el6.x86_64 #1 SMP Tue Dec 27 17:42:53 EST 2016 x86_64',
  'php-version' => '7.0.14',
  'php-sapi' => 'litespeed',
  'git-version' => '1.7.1',
  'git-full-path' => '/usr/bin/git
',
)
Git
array (
  'git-binary-as-configured' => 'git',
  'git-available' => true,
  'git-version' => '1.7.1',
  'git-binary-as-called-by-vp' => 'git',
  'git-full-path' => '/usr/bin/git
',
  'versionpress-min-required-version' => '1.9',
  'matches-min-required-version' => false,
)
```

**Comment by borekb** [comment]
It's not a VersionPress issue per se but I agree that people might encounter this. Your shell is different from what the webserver user might use.

BTW your environment doesn't seem to match the minimum required Git version.

**Comment by LorincJuraj** [comment]
> BTW your environment doesn't seem to match the minimum required Git 

Unfortunately on my hosting was this version installed, so I have to add additional installation of git. VersionPress just didn't know about my instalation, but I have fixed it by running
```wp vp config VP_GIT_BINARY '/home/myusername/bin/git/git'```

And when doing this I got an idea. What about adding another option to `wp vp config` - an option to configure path to WP so people who encounter this issue will not need to modify plugin code.
